Doc enhancements based on AR feedback (#97)
Esri/arcgis-runtime-toolkit-ios: README.md

The ArcGIS Runtime SDK for iOS Toolkit contains components that will simplify your iOS app development. Check out the [Examples](/Examples) project to see these components in action or read through the [Documentation](/Documentation) to learn more about them.

To use Toolkit in your project:

* **[Install with CocoaPods](#cocoapods)** - Add `pod ArcGIS-Runtime-Toolkit-iOS` to your podfile
* **[Install with Carthage](#carthage)** - Add `github "esri/arcgis-runtime-toolkit-ios"` to your cartfile
* **[Build manually](#manual)** - Build and include manually if you'd like to customize or extend toolkit

## Toolkit Components

* **[Augmented reality (AR)](Documentation/AR)** - Integrates the scene view with ARKit to enable augmented reality (AR).
* **[Bookmarks](Documentation/Bookmarks)** - Shows bookmarks, from a map, scene, or a list.
* **[Compass](Documentation/Compass)** - Shows a compass direction when the map is rotated. Auto-hides when the map points north up.
* **[JobManager](Documentation/JobManager)** - Suspends and resumes ArcGIS Runtime tasks when the app is background, terminated, and relaunched.

> * **[LegendViewController](Documentation/LegendViewController)** - Displays a legend for a single layer in your map (and optionally for its sub layers).

Reviewer: ```suggestion
* **[LegendViewController](Documentation/LegendViewController)** - Displays a legend for all the layers in a map or scene contained in an `AGSGeoView`.
```
